src/main.py

```diff
@@ -152,5 +152,4 @@ def run():
     print(constants.famTable)
 
 
-# Uncomment me for debugging!!
-# run()
+run()
```

Comment:
Should this be commented or uncommented?

Reply:
@PaulGurman Given that pytest still worked. I think we are in the all clear - i even updated README to display that

Reply:
@mikedelgaudio Cool, looks good to me then!